forte/integrals/integrals.cc

```diff
@@ -181,6 +181,8 @@ const psi::Dimension& ForteIntegrals::ncmopi() const { return ncmopi_; }
 
 size_t ForteIntegrals::ncmo() const { return ncmo_; }
 
+std::vector<size_t> ForteIntegrals::cmotomo() const { return cmotomo_; }
```

Comment:
```suggestion
const std::vector<size_t>& ForteIntegrals::cmotomo() const { return cmotomo_; }
```

Reply:
I like the fact that by doing it this way we either just pass a pointer or give the options to copy the data.